dist/edu-benefits-schema.json

```diff
@@ -365,62 +381,25 @@
         }
       }
     },
-    "previouslyFiledClaimWithVa": {
```

Comment:
These "previous claim" fields exist in the Spool file - we should follow up and see if they need to be collected.